docs/src/content/docs/configuration.mdx

```diff
@@ -81,6 +81,28 @@ An array of optional links to add to the `llms.txt` entrypoint.
 URLs provided here can be skipped by the LLM if a shorter context is needed.
 Use it for secondary information which is not already in your docs content.
 
+### `promote`
+
+**Type:** `string[]`  
+**Default:** `['index*', 'getting-started*', '!*/*']`
+
+Micromatch expressions to match page IDs that should be sorted to the top of the output.
+
+The default value promotes the pages `index` and `getting-started` to the top of the list, followed by any pages in the root directory of the docs.
```

Comment:
Hmm, not 100% sure about promoting all root-level pages. The thinking of the current sort is to aim to group related pages, so e.g. `/examples`, `/examples/x`, and `/examples/y` are together.

`getting-started` also feels a bit arbitrary? There’s no guarantee a site has such a page, so it feels like a somewhat opinionated choice (compared to `index` which is a standardised thing).